public/app/plugins/datasource/cloudwatch/components/Forms.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { InputHTMLAttributes, FunctionComponent } from 'react';
+import { Input, FormLabel } from '@grafana/ui';
+
+export interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  label: string;
+  inputEl?: React.ReactNode;
+  tooltip?: string;
+}
+
+export const FormField: FunctionComponent<Partial<Props>> = ({ label, inputEl, className, tooltip, ...inputProps }) => (
+  <>
+    <FormLabel width={8} className="query-keyword" tooltip={tooltip}>
+      {label}
+    </FormLabel>
+
+    {inputEl || <Input type="text" className={`gf-form-input`} {...inputProps} />}
```

Comment:
Mot sure I like this inputEl and that FormField has props from HTMLInputElement but you can change the inputEl to anything. 

(I think this is a big problem with the FormField in @grafana/ui as well and we are going to change that). 

Think it's better to let FormField just take label & tooltip as props and let inputEl be handled via children prop instead.

Reply:
I agree!